index.html

```diff
@@ -1,46 +1,176 @@
-        <div class="title">
+        <div class="head">
             <h2>Hey There.</h2>
-            <h2>I'm Rachid</h2> 
+            <h2>I'm Rachid</h2>
         </div>
         <p class="secondary">I'm a software developer</p>
-        <p class="details">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        <p class="details">I can help you build a product , feature or website Look through some of my work and
+            experience! If you like what you see and have a project you need coded,
             don’t hestiate to contact me.
-        
+
         </p>
         <div class="social-media">
-            <a href="https://github.com/benwmx" rel="noopener noreferrer" target="_blank"><img src="./img/002-github-big-logo.png" alt="the image ogf github icon"></a>
-            <a href="https://www.linkedin.com/in/rachid-boudaoudi-1621a0183/" rel="noopener noreferrer" target="_blank"><img src="./img/001-linkedin.png" alt="the image of linkedin icon"></a>
-            <a href="https://twitter.com/benwmx" rel="noopener noreferrer" target="_blank"><img src="./img/003-twitter.png" alt="the image of twitter icon"></a>
+            <a href="https://github.com/benwmx" rel="noopener noreferrer" target="_blank"><img
+                    src="./img/002-github-big-logo.png" alt="the image ogf github icon"></a>
+            <a href="https://www.linkedin.com/in/rachid-boudaoudi-1621a0183/" rel="noopener noreferrer"
+                target="_blank"><img src="./img/001-linkedin.png" alt="the image of linkedin icon"></a>
+            <a href="https://twitter.com/benwmx" rel="noopener noreferrer" target="_blank"><img
+                    src="./img/003-twitter.png" alt="the image of twitter icon"></a>
         </div>
     </section>
-       
-    
+
+
+    <section class="work">
+        <h1>My Recent Works</h1>
+        <div class="line1">
+        </div>
+        <div class="card one">
+            <h1 class="title">Professional Art <br>Printing Data</h1>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard</p>
+            <div class="languages-built">
+                <p>Html</p>
+                <p>Bootstrap</p>
+                <p>Ruby</p>
+            </div>
+            <button type="button" class="button project">
+                See Project
+            </button>
+        </div>
+        <div class="card two">
+            <h1 class="title">Professional Art <br>Printing Data</h1>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                industry's standard</p>
+            <div class="languages-built">
+                <p>Html</p>
+                <p>Bootstrap</p>
+                <p>Ruby</p>
+            </div>
+            <button type="button" class="button project">
```

Comment:
- [ ] This button is intended to provide a link. Please ensure that the button has the link tag `a` attached, and should follow the button styling: Active, Hover, and Pressed.